services/filesstore/s3store_test.go

```diff
@@ -7,26 +7,24 @@ import (
 	"testing"
 
 	"github.com/mattermost/mattermost-server/model"
+	"github.com/stretchr/testify/require"
 )
 
 func TestCheckMandatoryS3Fields(t *testing.T) {
 	cfg := model.FileSettings{}
 
 	err := CheckMandatoryS3Fields(&cfg)
-	if err == nil || err.Message != "api.admin.test_s3.missing_s3_bucket" {
-		t.Fatal("should've failed with missing s3 bucket")
-	}
+	res := err == nil || err.Message != "api.admin.test_s3.missing_s3_bucket"
```

Comment:
You can replace it using `require.NotNil(t, err)` and `require.Equal(t, err.Message, "api.admin.test_s3.missing_s3_bucket")`

Reply:
I changed this one according to your suggestion.